ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
@seandst I don't have an opinion regarding leaving it documented or not. So whatever you think is better works for me.

The difference is that this is for pulp smash and not for pulp itself. With this value you tell Pulp Smash what Pulp server version you are testing against. Pulp Smash is the automation framework used to test Pulp.